src/css/normalize.css

```diff
@@ -0,0 +1,351 @@
+/*! normalize.css v8.0.1 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Wonder if we'd combine these styles with the ones in `reset.css`, so we don't have two files doing essentially the same thing?

Reply:
This is a file that we got externally though, so if we ever want to update it with the new version it would be easier to keep it separate. Plus Grafana also has this separation so w probably should keep it the same 🤷

Reply:
> Plus Grafana also has this separation so w probably should keep it the same

Didn't know that. In that case this approach makes sense.